24_clustering.Rmd

```diff
@@ -78,43 +74,15 @@ plotReducedDim(tse, "MDS", colour_by = "clusters")
 To change the clustering parameters, simply change the parameters in 
 `HclustParam` function. To see the different parameters, check the 
 [HclustParam documentation](https://rdrr.io/github/LTLA/bluster/man/HclustParam-class.html).
-
-```{r bluster_sample_hclust2}
-# More complex use of the algorithm with another method and cut height.
-# We also add the full parameter to get the full hierarchical clustering
-# information to plot the dendrogram
-hclust.out <- clusterRows(x, HclustParam(method = "complete"), full = TRUE)
-
-# Add data to the TSE
-colData(tse)$clusters <- hclust.out$clusters
-
-# Get the dendrogram object
-dendro <- as.dendrogram(hclust.out$objects$hclust)
-
-# Plot the dendrogram
-plot(dendro)
-```
-
-Alternatively, it's also possible to use the `clusterCells` function from 
-the *scran* package.This function uses directly the tree as a parameter 
-instead of the assay itself. Here, we will use the PAM algorithm.
-
-```{r scran_sample_clust}
-library(scran)
-pam.out <- clusterCells(tse,
-  assay.type = "relabundance",
-  BLUSPARAM = PamParam(centers = 5)
-)
-```
+Check dedicated part to see other examples.
 
 ### Taxa clustering
```

Comment:
Remove "Taxa clustering" section. The chapter 6.3 discusses those things. You can briefly refer to that section possibly, but that is not critical. Let us avoid duplicating info.